src/map-marker/__tests__/floating-marker.test.js

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+/* eslint-env browser */
+
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {FloatingMarker} from '../index.js';
+import {FLOATING_MARKER_SIZES} from '../constants.js';
+
+describe('Floating Marker', () => {
+  const label = 'test';
+  Object.values(FLOATING_MARKER_SIZES).forEach(
+    //$FlowFixMe
+    (size: FloatingMarkerSizeT) => {
+      test(`renders floating marker with size ${size}`, () => {
+        const {container} = render(
+          <FloatingMarker label={label} size={size} />,
+        );
+        const marker = container.querySelector('[data-baseweb="map-marker"]');
+        expect(marker).not.toBeNull();
+      });
+    },
+  );
+});
```

Comment:
This test can probably be deleted. We take screenshots of all of the .scenario files and those do a better job at asserting rendering

Reply:
Ok. I will also delete `fixed-marker.test.js` as well.